### PR TITLE
AETHER-139 - handover test on staging to Nucleus

### PR DESCRIPTION
**Pull request by thakurajayL**
Handover fixes for adding new state to handle enb transfer message.
and also fixing decoding of handover messages.

### CODE REVIEW

**Comment by thakurajayL** [comment]
@badhrinathpa @asreek can you please review code.

File: scripts/SMCodeGen/codeGen.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
 import json, sys, os
 from os.path import join
 import utils
+#set path where you have Template-Toolkit 
+#sys.path.append("/users/ajayonf/Nucleus.Ajay/scripts/SMCodeGen/Template-Toolkit-Python")
```
Comment (asreek) [inline scripts/SMCodeGen/codeGen.py:21]: Remove the commented code if not required.

**Comment by yoooou** [comment]
retest this please

**Comment by sindack** [comment]
retest this please

**Comment by yoooou** [comment]
retest this please

**Comment by yoooou** [comment]
retest this please

**Comment by yoooou** [comment]
retest this please

**Comment by yoooou** [comment]
retest this please

**Pull request merged**